charts/sn-platform-slim/templates/broker/broker-cluster.yaml

```diff
@@ -40,6 +40,7 @@ metadata:
 {{- end }}
 spec:
   zkServers: "{{ template "pulsar.zookeeper.connect" . }}{{ .Values.metadataPrefix }}"
+  configurationStoreServers: "{{ template "pulsar.configurationStore.connect" . }}{{ .Values.metadataPrefix }}"
```

Comment:
I saw the parameter `` is a deprecated parameter in this [2.10.x doc]( https://pulsar.apache.org/docs/2.10.x/reference-configuration/), shall we consider supporting the new parameter instead?